sdk/confidentialledger/azure-confidentialledger/README.md

```diff
@@ -300,77 +334,63 @@ Async clients should be closed when they're no longer needed. These objects are
 
 ```python
 from azure.identity.aio import DefaultAzureCredential
-from azure.confidentialledger.aio import ConfidentialLedgerClient
-from azure.confidentialledger.identity_service.aio import ConfidentialLedgerIdentityServiceClient
+from azure.confidentialledger.aio import (
+    ConfidentialLedgerClient,
+)
+from azure.confidentialledger_identity_service.aio import (
+    ConfidentialLedgerIdentityServiceClient,
+)
 
-identity_client = ConfidentialLedgerIdentityServiceClient("https://identity.accledger.azure.com")
-network_identity = await identity_client.get_ledger_identity(
+identity_client = ConfidentialLedgerIdentityServiceClient("https://identity.confidential-ledger.core.azure.com")
```

Comment:
unless there's a special need (i.e. this service has special async behavior), we generally just link to async samples in the samples folder, instead of including an async example in the readme